app/views.py

```diff
@@ -0,0 +1,55 @@
+from django.shortcuts import render, redirect
+from .models import Url
+from djongo import models
+from bson.objectid import ObjectId
+from .forms import NewUrlForm
+from django.contrib import messages
+
+# Create your views here.
+def parseObjIds(a):
+    return {
+        "url": a['url'],
+        "mongo_id": a['_id']
+        }
+
+def urls(req):
+    if req.method == 'POST':
+        url = NewUrlForm(req.POST)
+        newUrl = url.save()
+        response = redirect("urls-show", id=newUrl._id)
+        return response
+    else:
+        urls = list(Url.objects.all().values('url', '_id'))
```

Comment:
Cool to see exsiting urls from the database automatically. May potentially get too long if you deployed it?